### DISCUSSION TITLE
[Feature Request] Custom map layers

### DISCUSSION THREAD

**Discussion by Kiriyashiki** [Freika/dawarich#715] [Ideas]
Currently it is only possible to use the pre defined map layers. It would be nice if we could add our own map layers with a configuration file or something.
Would be useful for those who self host a tile server or want another source added.

**Comment by marsmathis** [comment]
Seconded. I would love to use my self-hosted tile server (because my tiles load faster, they are vector tiles, and they look nicer).

**Comment by Freika** [comment]
Done in 0.24.1 (only raster maps for now)

**Reply by jpbaril** [reply]
@Freika Does that mean we can add back Stadia Maps (provided I created a Stadia account) ?
If so, how ?

**Reply by jpbaril** [reply]
Ok, that was easy:
`https://tiles.stadiamaps.com/tiles/alidade_smooth/{z}/{x}/{y}{r}.png`
